lib/platform/azure/index.ts

```diff
@@ -520,13 +521,13 @@ export async function ensureCommentRemoval(
   }
 }
 
-export function setBranchStatus(
-  branchName: string,
-  context: string,
-  description: string,
-  state: string,
-  targetUrl: string
-): void {
+export function setBranchStatus({
+  branchName,
+  context,
+  description,
+  state,
+  url: targetUrl,
```

Comment:
@ViceIce shouldn't we ideally pick one name (e.g. `targetUrl`) and then use it uniformly in both platform as well as worker code?

Reply:
@rarkins Yes, but i think @souravdasslg want to make as less changes as possible. So i think we should cleanup code in a later pr?

Reply:
ok then, good idea